### PR TITLE
Data: Iyanu1396

### PR DESCRIPTION
**Pull request by Iyanu1396**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4107"></a>

### CODE REVIEW

File: data/Iyanu1396.json [hunk 1]
```diff
@@ -2,8 +2,15 @@
   "name": "IyanuOluwa",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Frontend Developer | Javascript ",
+  "bio": "Frontend Developer | **Javascript** ",
   "avatar": "https://github.com/Iyanu1396.png",
+"socials": [
+  { "icon": "FaTwitter", "url": "https://twitter.com/iyanu_codes" },
+  { "icon": "FaGithub", "url": "https://github.com/Iyanu1396" },
```
Comment (swagataroy30) [inline data/Iyanu1396.json:9]: remove comma from line 9

File: data/Iyanu1396.json [hunk 2]
```diff
@@ -2,8 +2,15 @@
...
+],
+  "tags": [
+    "Javascript",
```
Comment (swagataroy30) [inline data/Iyanu1396.json:12]: remove comma from line 12

File: data/Iyanu1396.json (hunk 1 above)
Comment (Iyanu1396) [reply data/Iyanu1396.json]: Done

File: data/Iyanu1396.json (hunk 2 above)
Comment (Iyanu1396) [reply data/Iyanu1396.json]: Done

**Review by iamthecloverly — APPROVED** [review]

**Review by swagataroy30 — APPROVED** [review]
LGTM !

**Comment by Iyanu1396** [comment]
> LGTM !

Waiting for it to be merge

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 👍

**Pull request merged**